test/selenium/web_app_test.js

```diff
@@ -0,0 +1,48 @@
+
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict';
+
+var webdriver = require('selenium-webdriver');
+
+var HOST_URL = 'https://fxbox.github.io/app';
+
+var SetUpWebapp = require('./lib/setup_webapp.js');
+var webAppMainPage;
+var setUpWebapp;
+
+describe('UI app', function() {
+  var driver;
+  this.timeout(8000);
+
+  //const PASS = '12345678';
+
+  before(function() {
+    driver = new webdriver.Builder().
+      forBrowser('firefox').
+      build();
+  });
+
+  beforeEach(function() {
+    driver.get(HOST_URL);
+  });
+
+  after(function() {
+    driver.quit();
+  });
+     
+  describe('open the web app', function() {
+
+    beforeEach(function() {
+          setUpWebapp = new SetUpWebapp(driver);
+          webAppMainPage = setUpWebapp.getAppMainView();
+          return webAppMainPage;
+        });
+
+    it('should log in from web app', function() {
+      return webAppMainPage.connectToFoxBox().then((logInView) => 
+        logInView.loginSuccess(12345678));
```

Comment:
It is set in the sessions_ui_test.js.  Both tests are dependant, which is not the best, first the set up and then the log in in the app. Thought about other options but not sure which is the best to proceed without modifing all the tests we already have